### PR TITLE
Added Android Toasts

### PR DESCRIPTION
**Pull request by Jarrio**
Added the toasts alert. 

image

### CODE REVIEW

File: src/react/native/api/ToastAndroid.hx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+@:jsRequire('react-native', 'ToastAndroid')
+extern class ToastAndroid {
+	static function show(message:String, duration:Duration):Void;
+	static function showWithGravity(message:String, duration:Gravity, gravity:Float):Void;
+}
+
+@:enum
+abstract Duration(Int) {
+    var SHORT = 0;
```
Comment (kevinresol) [inline src/react/native/api/ToastAndroid.hx]: I suspects this values are wrong:
https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/modules/toast/ToastModule.java#L31-L36

File: src/react/native/api/ToastAndroid.hx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package react.native.api;
+
+@:jsRequire('react-native', 'ToastAndroid')
+extern class ToastAndroid {
+	static function show(message:String, duration:Duration):Void;
+	static function showWithGravity(message:String, duration:Gravity, gravity:Float):Void;
```
Comment (kevinresol) [inline src/react/native/api/ToastAndroid.hx:6]: `duration:Gravity`???

File: src/react/native/api/ToastAndroid.hx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+@:enum
+abstract Duration(Int) {
+    var SHORT = 0;
+    var LONG = 1;
+}
+
+@:enum
+abstract Gravity(Int) {
+    var TOP = 0;
```
Comment (kevinresol) [inline src/react/native/api/ToastAndroid.hx:17]: Not sure about these values:
https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/modules/toast/ToastModule.java#L50-L54

**Comment by Jarrio** [comment]
oops! I adjusted the values in [#992a76e](https://github.com/Jarrio/haxe-react-native/blob/master/src/react/native/api/ToastAndroid.hx) It seems to work fine with the new values! Gif attached below to demonstrate 

gif

**Comment by kevinresol** [comment]
I added this manually, see: c61f860
Using constants from the API is more safe.
Also make it more similar to the js usage.

**Comment by Jarrio** [comment]
I'm still new to the whole creating externs thing so I wasn't aware of being able to do that. Neat!

**Pull request closed without merging**